cvat-canvas/README.md

```diff
@@ -0,0 +1,93 @@
+## Module
+
+- Written on typescript
+- Contains the class ```Canvas```
+
+## Creation
+Canvas is created by using constructor:
+
+```js
+    const canvas = new Canvas();
+```
+
+- Canvas has transparent background
+
+Canvas itself handles:
+- Shape context menu (PKM)
+- Image moving (mousedrag)
+- Image resizing (mousewheel)
+- Image fit (dblclick)
+- Remove point (PKM)
+- Polyshape editing (Shift + LKM)
+
+## API
+### Methods
+
+All methods are sync.
+
+```js
+    html() => canvas HTML element
+    setup(const FrameData: frameData, [{
+        state: ObjectState,
```

Comment:
Probably it is better to guarantee ids for objects inside cvat-canvas.js. For example, cvat_canvas_{state.id}_... Thus everybody will be able to change your default styles in any way just generate right css. I don't think that providing here the limited API will work good. Do reasonable job inside cvat-canvas.js but give a way to change appearance of shapes as UI wants.  Also you can "publish" css class names for your elements. Thus you can redefine them as well.

Reply:
@nmanovic 
Done.